po/de.po

```diff
@@ -3767,7 +3766,7 @@ msgstr "Es ist wahrscheinlich, dass er gegen Ihre Stellung gerichtet ist."
 
 #: data/base/sequenceaudio/cam2/c2diif2.txt:4
 msgid "You are to start the immediate evacuation of Beta Base to the safe haven."
-msgstr "Beginnen Sie sofort damit die Betabasis in die Sicherheitszone zu evakuieren."
+msgstr "Beginnen Sie sofort damit die Betabasis in den sicheren Hafen zu evakuieren."
```

Comment:
This sounds a bit odd to me. The preposition "in" does not work smoothly with "den sicheren Hafen", so should probably be "zum sicheren Hafen".

Reply:
Okay.